src/uwtools/drivers/driver.py

```diff
@@ -85,7 +85,7 @@ def _run_via_local_execution(self):
         A run executed directly on the local system.
         """
         yield self._taskname("run via local execution")
-        path = self._rundir / "done"
+        path = self._rundir / f"done.{self._driver_name}"
```

Comment:
This follows the example set for `runscript` and allows `done` files for multiple drivers to potentially coexist in the same run directory.